src/components/modules/header/HeaderAusMomentSupporter.tsx

```diff
@@ -0,0 +1,145 @@
+    color: ${brandText.primary};
+    margin-bottom: ${space[2]}px;
+    line-height: 1.15;
+
+    ${from.desktop} {
+        ${textSans.medium()};
+        margin-bottom: ${space[1]}px;
+    }
+`;
+
+const headerYellowHighlight = css`
+    color: ${brandAlt[400]};
+    font-weight: 700;
+    margin: 5px 0;
+`;
+
+const ausMapLinkStyles = css`
+    ${textSans.small()};
+    color: ${brandText.primary};
+    line-height: 1.15;
+
+    ${from.desktop} {
+        ${textSans.medium()};
+    }
+`;
+
+const mobileCtaStyles = css`
+    ${until.mobileMedium} {
+        display: none;
+    }
+
+    ${textSans.xxsmall()}
+    color: ${brandAlt[400]};
+`;
+
+const linkStyles = css`
+    height: 24px;
+    min-height: 24px;
+    ${textSans.small({ fontWeight: 'bold' })};
+    border-radius: 16px;
+    padding: 0 ${space[3]}px;
+    margin-right: 10px;
+    margin-bottom: 6px;
+
+    ${from.desktop} {
+        ${textSans.medium({ fontWeight: 'bold' })};
+        height: 36px;
+        min-height: 36px;
+        padding: 0 ${space[4]}px;
+        border-radius: 36px;
+    }
+
+    svg {
+        width: 24px;
+    }
+`;
+
+const Header: React.FC<HeaderRenderProps> = (props: HeaderRenderProps) => {
+    const { heading, primaryCta } = props.content;
+
+    const numberOfSupporters = useNumberOfSupporters();
+
+    return (
+        <div>
+            <Hide below="mobileMedium">
+                <div>
+                    <div css={ausMomentHeadingStyles}>{heading}</div>
+                </div>
+            </Hide>
+            <Hide below="tablet">
+                <div css={ausMomentSubheadingStyles}>
+                    You&apos;re one of{' '}
+                    <Link
+                        href="https://support.theguardian.com/aus-map?INTCMP"
+                        css={ausMapLinkStyles}
+                    >
+                        <span css={headerYellowHighlight}>{numberOfSupporters} </span>
+                        supporters in Australia
+                    </Link>
+                </div>
+            </Hide>
+
+            {primaryCta && (
+                <>
+                    <ThemeProvider theme={buttonReaderRevenueBrand}>
+                        <Hide below="tablet">
+                            <LinkButton
+                                priority="primary"
+                                href={primaryCta.ctaUrl}
+                                icon={<SvgArrowRightStraight />}
+                                iconSide="right"
+                                nudgeIcon={true}
+                                css={linkStyles}
+                            >
+                                {primaryCta.ctaText}
+                            </LinkButton>
+                        </Hide>
+                    </ThemeProvider>
+                </>
+            )}
+            <Hide above="tablet">
+                <div>
+                    <Link href="http://support.theguardian.com/contribute" css={mobileCtaStyles}>
```

Comment:
Should I add a customised tracking link here or would it be done by the test?

Reply:
yep, `primaryCta.ctaUrl`

Reply:
Aha! Would it be like this? `<Link href={primaryCta.ctaUrl} ....>`

Reply:
that's right